### PR TITLE
Update GETTING_STARTED.md

### PR DESCRIPTION
**Pull request by gpolumbo-broad**
Path to Artifactory Password in Vault changed at some point and the docs were not updated to reflect that new path.

No Jira because this is just a change to documentation, not actual code.

**PR checklist**
- [ ] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [ ] Bump the version in `project/Settings.scala` `createVersion()`
- [ ] Update `CHANGELOG.md` for those libraries
- [ ] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [ ] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

### CODE REVIEW

**Review by rtitle — APPROVED** [review]
:shipit:

File: GETTING_STARTED.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Before starting, set artifactory credentials in your environment:
 
 ```
 export ARTIFACTORY_USERNAME=dsdejenkins
-export ARTIFACTORY_PASSWORD=$(docker run -v $HOME:/root --rm broadinstitute/dsde-toolbox:dev vault read -field=password secret/dsde/firecloud/common/artifactory/dsdejenkins)
+export ARTIFACTORY_PASSWORD=$(docker run -v $HOME:/root --rm broadinstitute/dsde-toolbox:dev vault read -field=password secret/dsp/accts/artifactory/dsdejenkins)
```
Comment (andy7i) [inline GETTING_STARTED.md:11]: I tried running the old command and the new command. Looks like both work, but I get 2 different values. Is the old value invalid now?

**Review by andy7i — APPROVED** [review]
👍 
1 question but not blocking.

File: GETTING_STARTED.md (hunk 1 above)
Comment (rtitle) [reply GETTING_STARTED.md:11]: Yeah the artifactory password was changed, the old vault path is incorrect

Comment (andy7i) [reply GETTING_STARTED.md:11]: oh I see. In that case, it might be helpful to delete the old secret as well!

**Pull request merged**